### DISCUSSION TITLE
Icons list is not loading

### DISCUSSION THREAD

**Discussion by mehmood-v** [bgrand-ch/grapesjs-icons#6] [General]
![image](https://github.com/bgrand-ch/grapesjs-icons/assets/95611144/13697516-9d6e-4b73-a249-ec024b8f2565)

**Comment by mehmood-v** [comment]
I have integrated the package with my grapes js application according to your documentation but the icons are not being loaded in the list. Could you please help.

**Comment by bgrand-ch** [comment]
Hello, could you show me the configuration of the `grapesjs-icons` plugin please?

You should use the abbreviation of the icon library. The full name will be implemented soon.

Example with "Remix icons" (Remix = "ri" in URL):
https://icon-sets.iconify.design/ri/

![Screenshot 2023-10-23 at 12 59 38](https://github.com/bgrand-ch/grapesjs-icons/assets/17108974/f724c197-ade1-418c-a9be-fcaf83157775)

**Reply by mehmood-v** [reply]
const options = {
    // see https://icon-sets.iconify.design/
    collections: [
      "ri", // Remix Icon by Remix Design
      "mdi", // Material Design Icons by Pictogrammers
      "uim", // Unicons Monochrome by Iconscout
      "streamline-emojis", // Streamline Emojis by Streamline
      "material-symbols",
    ],
  };


these are the same configuration as your documentation. I haven't changed anything.

**Comment by bgrand-ch** [comment]
@mehmood-v Hello, I'm going to test and fix the plugin if necessary over the next few days. I'll let you know here when it's OK.

**Comment by bgrand-ch** [comment]
@mehmood-v Do you have an error in the DevTools console? 🤔

**Comment by bgrand-ch** [comment]
@mehmood-v ℹ️ WIP #8
